Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    internal class CredentialsManager
+    {
+        public struct Person
+        {
+            public Person(string name, string password)
+            {
+                Name = name;
+                Password = password;
+            }
+
+            public string Name { get; set; }
+            public string Password { get; set; }
+        }
+
+        internal static void Demo()
+        {
+            Console.WriteLine("Would you like to register or login");
+            Console.WriteLine("Choose R voor register");
+            Console.WriteLine("Choose L voor login");
+            var choice = Console.ReadLine();
+            switch (choice)
+            {
+                case "R":
+                    Register();
+                    break;
+
+                case "L":
+                    Login();
+                    break;
+
+                default:
+                    Console.WriteLine("This is not a right choice. Only R and L are valid");
+                    break;
+            }
+        }
+
+        private static void Login()
+        {
+            var isValidLogin = false;
+            var data = File.ReadAllText(@"Database/user.txt");
+            var fileData = data.Split($"{Environment.NewLine}");
+            var input = InputData();
+            bool isValidInput = TryParse(input, out Person personData);
+             if(!isValidInput)
+            {
+                Console.WriteLine("You entered invalid data");
+                return; 
+            }
+            for (int i = 0; i < 1; i++)
+            {
+                var person = fileData[i];
+                var splitted = person.Split(";");
+                if (splitted[1] == personData.Password & splitted[0] == personData.Name)
+                {
+                    isValidLogin = true;
+                }
+            }
+
+            Console.WriteLine(isValidLogin);
+        }
+
+        private static bool TryParse(string input, out Person person)
```

Comment:
Try parse should be a part of the class that you are applying it for. TryParse(string, out int) is a part of int; TryParse(string, out Person) should be a part of Person (Credentials)

Reply:
I hope I understand you well and solved it the right way